.circleci/config.yml

```diff
@@ -664,7 +696,7 @@ workflows:
             - eunit
             - static_analysis
             - linux_package
-            - hodl
+            - hodl_blue
```

Comment:
As blue is deployed before green, this means that Ubuntu binary uploaded before green deployed. Ok. (Just confirming.)